x-pack/plugins/fleet/common/types/models/epm.ts

```diff
@@ -579,12 +579,21 @@ export interface StateContext<T> {
   latestExecutedState?: LatestExecutedState<T>;
 }
 
+export interface EsIndexPattern {
+  /** The user-facing title */
```

Comment:
nit: I am not sure the field names are too logical, maybe renaming `title` to `name` and `name` to `pattern` would make more sense?

Reply:
Open to suggestions, I aligned with what was already defined here: https://github.com/elastic/kibana/blob/495d7b5b681efa4b7cfb9ff8cb68495a64f757d2/x-pack/plugins/fleet/server/services/epm/packages/get.ts#L388

We have three things: The pattern, the technical name and the user readable name. I agree that `pattern`, `name` and `display_name` is an understandable naming scheme

Reply:
I see, that `title` field name is not used anywhere else, so I think it's good to rename.